miniscope_io/stream_daq.py

```diff
@@ -455,8 +457,11 @@ def _format_frame(
         try:
             for frame_data, header_list in exact_iter(frame_buffer_queue.get, None):
 
-                locallogs.debug("Found frame in queue")
+                if not frame_data:
+                    imagearray.put((None, header_list))
+                    continue
                 if len(frame_data) == 0:
+                    imagearray.put((None, header_list))
```

Comment:
Just a note that the test is modified because these lines squeeze out a bit more header data. But we should update this test itself soon as @sneakers-the-rat mentioned.